src/Wordpress/PluginActions.php

```diff
@@ -52,4 +52,65 @@ public function loginWordPress()
 
         return $response;
     }
+
+    /**
+     * GET /plugin/:zonedId/settings.
+     *
+     * @return mixed
+     */
+    public function getPluginSettings()
+    {
+        // Always return true
+        $value = true;//$this->dataStore->getIpRewrite();
+
+        $response = $this->api->createAPISuccessResponse(
+            array(
+                array(
+                'id' => 'ip_rewrite',
+                'value' => $value,
+                'editable' => true,
+                'modified_on' => '',
+                ),
+            )
+        );
+
+        $response['errors'] = []; // TODO: This doesn't seem a nice way
+        $response['messages'] = [];
+
```

Comment:
Correct - this isn't the ideal way.  Lets add these fields as part of `createAPISuccessResponse()` since every response will need empty arrays for errors and messages.